src/components/visittkort/Visittkort.tsx

```diff
@@ -1,17 +1,18 @@
 import * as React from 'react';
-import {Person} from "../../models/person";
+import { Person } from '../../models/person';
+import Innholdstittel from 'nav-frontend-typografi/lib/innholdstittel';
 
 interface VisittkortProps {
-    person: Person
+    person: Person;
 }
 
-function Visittkort({person} : VisittkortProps) {
+function Visittkort({person}: VisittkortProps) {
     return (
         <div>
-            <h1>{person.fornavn} {person.etternavn}</h1>
+            <Innholdstittel>{person.fornavn} {person.etternavn}</Innholdstittel>
```

Comment:
Hører dette hjemme i denne PR'en? 🤔

Reply:
Ja, det er egentlig bare for eksemplets skyld. En slags placeholder :)

Reply:
Aha, I see